src/ng-xform/modules-import.ts

```diff
@@ -0,0 +1,7 @@
+import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
+import { BsDatepickerModule } from 'ngx-bootstrap/datepicker';
+import { ModuleWithProviders } from '@angular/core';
+
+export const bsDatepickerModuleWithProviders: ModuleWithProviders = BsDatepickerModule.forRoot();
```

Comment:
Docs on why you need to do that. Add links for Github issues with the discussion of not calling methods on external modules imports